tutorials/README.md

```diff
@@ -185,16 +190,16 @@ There are many legitimate app UX circumstance, that can dictate the dynamicness
 
 ##### _Tutorial Sample:_
 
-> [Hooking Up Next Video](Tutorials)
+> [Hooking Up Next Video](/tutorials)
 
 <a name="tutorial-4"></a>
-## Tutorial 4: Error Handling in the SDK
+### Tutorial 4: Error Handling in the SDK
 
-This tutorial sample shows you how to handle various different types of errors that can occur when using the OVPSDK and how to catch and identify them. How you handle these in your app is up to you. The SDK is designed to either return a valid SDK or Player instance otherwise it returns an error. There is no middle ground. If you don’t get a valid instance, you should look at the error result instead to determine why. This section describes some common issues.
+This tutorial sample shows you how to handle various different types of errors that can occur when using the VVPSDK and how to catch and identify them. How you handle these in your app is up to you. The SDK is designed to either return a valid SDK or Player instance otherwise it returns an error. There is no middle ground. If you don’t get a valid instance, you should look at the error result instead to determine why. This section describes some common issues.
 
 #### SDK Initialization Errors
 
-For various reasons, the SDK may fail to initialize. The most common reason for this, is you’re trying to use the OVPSDK without first having [onboarded your app’s bundle ID](##Onboarding\ your\ Apps\ for\ SDK\ Authentication). In this case, you’ll get an error that looks like something like this:
+For various reasons, the SDK may fail to initialize. The most common reason for this, is you’re trying to use the VVPSDK without first having [onboarded your app’s bundle ID](##Onboarding\ your\ Apps\ for\ SDK\ Authentication). In this case, you’ll get an error that looks like something like this:
```

Comment:
`[onboarded your app’s bundle ID](##Onboarding\ your\ Apps\ for\ SDK\ Authentication)` - not valid!

Reply:
Can you please add a link to the main parent README that goes right to the Onboarding section @RomanTysiachnik